README.md

```diff
@@ -55,18 +55,9 @@ cd example/
 virtualenv -ppython3 env
 source env/bin/activate
 
-pip install --no-deps djangosaml2-spid
 pip install djangosaml2-spid
 ````
 
-⚠️ Why `pip install` have beed executed twice? spid-django needs a fork of PySAML2 that's not distribuited though pypi.
-This way to install it prevents the following error:
-
-````
-ERROR: Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.
-djangosaml2-spid depends on pysaml2@ git+https://github.com/peppelinux/pysaml2.git@pplnx-7.0.1#pysaml2
-````
-
```

Comment:
We should say something in the README about the monkey patching, the sources if pplnx's pysaml2 fork and where it is in the code, for a better big view of the critical changes we made.

Veery good Davide, very good.
That's something I can also adopt in SATOSA-Saml2Spid by default!

Reply:
> We should say something in the README about the monkey patching, the sources of pplnx's pysaml2 fork and where it is in the code, for a better big view of the critical changes we made.

I will update the PR with monkey patch info and adding changes from your [new PR for pysaml2](https://github.com/IdentityPython/pysaml2/pull/812).

Reply:
Already commited here
https://github.com/italia/spid-django/pull/82/commits/882d5b377137b9a86629affbb6b90149905a6484

Reply:
Also add a fix for logger in `_saml2.py`... 

Check if the monkey-patch alert info in `README.md` can be enough.